lua/astrocommunity/editing-support/stickybuf-nvim/init.lua

```diff
@@ -0,0 +1,5 @@
+return {
+  "stevearc/stickybuf.nvim",
+  opts = {},
+  event = { "User AstroFile", "InsertEnter" },
```

Comment:
Won't `User AstroFile` be enough?

Reply:
For comparasion, stevearc does:
`  { "stevearc/stickybuf.nvim", cmd = { "PinBuffer", "PinBuftype", "PinFiletype" }, opts = {} },
`